### ISSUE TITLE
A_Indirection(A_ArrayExpr) rendered incorrectly

### ISSUE THREAD

**Issue by df7cb** [lelit/pglast#99]
`select (array[1])[1]` gets rendered as `SELECT ARRAY[1][1]` which is invalid syntax:

pglast 3.8 on Debian Python 3.9.10:
```
>>> import pglast
>>> from pglast.stream import RawStream
>>> q = pglast.parse_sql('select (array[1])[1]')
>>> q
(<RawStmt stmt=<SelectStmt targetList=(<ResTarget val=<A_Indirection arg=<A_ArrayExpr elements=(<A_Const val=<Integer val=1>>,)> indirection=(<A_Indices is_slice=False uidx=<A_Const val=<Integer val=1>>>,)>>,) limitOption=<LimitOption.LIMIT_OPTION_DEFAULT: 0> op=<SetOperation.SETOP_NONE: 0> all=False> stmt_location=0 stmt_len=0>,)
>>> RawStream()(q)
'SELECT ARRAY[1][1]'

$ psql -c "select (array[1])[1]"
 array 
───────
     1

$ psql -c "SELECT ARRAY[1][1]"
FEHLER:  42601: Syntaxfehler bei »[«
ZEILE 1: SELECT ARRAY[1][1]
                        ^
ORT:  scanner_yyerror, scan.l:1176
```

**Comment by lelit** [comment]
Thank you, will try to fix as soon as possible.

**Comment by df7cb** [comment]
No need to rush, it's not blocking anything here, I just discovered it when playing with node types.

Thanks!

**Comment by lelit** [comment]
This too seems fixed in v3.

**Comment by df7cb** [comment]
Thanks, works now!

**Comment by lelit** [comment]
Released in v3.9.
